vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/crawler/LinkCrawler.java

```diff
@@ -115,12 +111,11 @@ public void onBeforeExit()
         linkCrawlerData.getAbsoluteUrls().removeAll(urlsWithInvalidRedirects);
     }
 
-    private boolean isExcluded(String path)
+    private boolean isExcluded(String url)
     {
-        String fileExtensionFromPath = getExtension(path).toLowerCase();
-
-        return !fileExtensionFromPath.isEmpty()
-                && excludeExtensionsPattern.map(p -> p.matcher(fileExtensionFromPath))
+        URI uri = UriUtils.createUri(url);
+        URI uriWithoutQueryAndFragment = UriUtils.removeFragment(UriUtils.removeQuery(uri));
```

Comment:
we need to indicate in the docs the fragment and URL query are ignored at filtering